inf-clojure.el

```diff
@@ -391,12 +384,20 @@ Fallback to `default-directory.' if not within a project."
           ((file-exists-p "build.boot") "boot")
           (t nil))))
 
+(defun inf-clojure-flavor ()
+  "Determine the flavor of the project."
+  (pcase inf-clojure-repl-flavor
+    (lumo (inf-clojure--lumo-repl-command))
+    (_ nil)))
```

Comment:
Are you talking about the start command specifically or all the commands to customize?
Because it is a bit inconvenient imho to carry around a huge `.dir-locals.el` from project to project for instance, the local var seems like a good compromise, let me send an addition I made.